lib/internal/child_process.js

```diff
@@ -361,19 +361,7 @@ function onErrorNT(self, err) {
 
 
 ChildProcess.prototype.kill = function(sig) {
-  var signal;
-
-  if (sig === 0) {
-    signal = 0;
-  } else if (!sig) {
-    signal = constants['SIGTERM'];
-  } else {
-    signal = constants[sig];
-  }
-
-  if (signal === undefined) {
-    throw new Error('Unknown signal: ' + sig);
-  }
+  const signal = convertToValidSignal(!sig && sig !== 0 ? 'SIGTERM' : sig);
```

Comment:
It falls back to `SIGTERM` now also, right? `!sig` will be `true` when `sig` is `undefined` and it is not zero, so `SIGTERM`.